src/senaite/patient/subscribers/analysisrequest.py

```diff
@@ -92,13 +93,38 @@ def update_patient(instance):
     if mrn is None:
         return
     patient = patient_api.get_patient_by_mrn(mrn, include_inactive=True)
+    # Create a new patient
     if patient is None:
-        logger.info("Creating new Patient with MRN #: {}".format(mrn))
+        if patient_api.is_patient_allowed_in_client():
+            # create the patient in the client
+            container = patient_api.get_patient_folder()
```

Comment:
This should be

```python
# create the patient in the client
container = instance.getClient()
```

Reply:
Thanks for catching this one! Fixed in 679f9a8